numpy/random/bit_generator.pyx

```diff
@@ -260,6 +260,7 @@ cdef class SeedSequence():
     ----------
     entropy : {None, int, sequence[int]}, optional
         The entropy for creating a `SeedSequence`.
+        note: all entropy not None must be non-negative.
```

Comment:
It isn't common to use `note`.  Just something along of the lines "All integer values must be non-negative."  Same below.